trashtalk/templates/_base.html

```diff
@@ -1,68 +1,84 @@
+<!DOCTYPE html>
+<!--
+Outline for rest of the webpage
+-->
+
 <html>
 {% load static %}
-<head>
-    <meta charset="windows-1252">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-{#    JQUERY #}
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
-{#    BOOTSTRAP #}
-    <link rel="stylesheet"
-          href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
-{#    DATE and TIME PICKER #}
-    <link rel="stylesheet" type="text/css" href={% static "stylesheets/datepicker.css" %}>
-    <script src="http://netdna.bootstrapcdn.com/bootstrap/3.1.1/js/bootstrap.min.js"></script>
-    <link rel="stylesheet"
-          type="text/css"
-          href={% static "stylesheets/bootstrap-timepicker.min.css" %}>
-    <script type="text/javascript"
-            src={% static "scripts/bootstrap-datepicker.js" %}></script>
-    <script type="text/javascript"
-            src={% static "scripts/bootstrap-timepicker.min.js" %}></script>
+    <head>
+        <meta charset="windows-1252">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <script src={% static "scripts/jquery.min.js" %}></script>
+        <link rel="stylesheet" href={% static "stylesheets/bootstrap.min.css" %}>
+        <script src={% static "scripts/bootstrap.min.js" %}></script>
 
-    <title>Trash Talk</title>
+        <link rel="stylesheet" type="text/css" href={% static "stylesheets/datepicker.css"%}>
+        <script type="text/javascript" src={% static "scripts/bootstrap-datepicker.js" %}></script>
+        <script type="text/javascript" src={% static "stylesheets/bootstrap-timepicker.min.css" %}></script>
+        <script type="text/javascript" src={% static "scripts/bootstrap-timepicker.min.js" %}></script>
+        <title>Trash Talk: {{section}}</title>
     </head>
+
     <body>
-{#    TODO: Issue #75: Export all inline styles to an external syltesheet. #}
-        <header style="text-align: center;">
-            <h1>Trash Talk</h1>
-        </header>
+        <header>
+             <nav class="navbar navbar-inverse">
+                  <div class="container-fluid">
+                    <div class="navbar-header">
+                      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                      </button>
+                        <!--TODO-->
+                      <a class="navbar-brand" href="{% url 'home' %}">TrashTalk</a>
+                    </div>
+
+                    <div class="collapse navbar-collapse" id="myNavbar">
+                        {%if current_user.authenticated%}
```

Comment:
I think `{% if user.is_authenticated %}` is what you mean here. See docs: https://docs.djangoproject.com/en/1.11/topics/auth/default/#all-authentication-views

The user object in our new django views is just `user` right now, not `current_user`